unpackaged/config/qa/objects/___NAMESPACE___DataImport__c.object

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <fields>
-        <fullName>custom_campaign__c</fullName>
+        <fullName>new_lookup_campaign__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
         <externalId>false</externalId>
         <inlineHelpText>Opportunity.custom_lookup__c</inlineHelpText>
-        <label>custom_campaign</label>
+        <label>new_lookup_campaign</label>
         <referenceTo>Campaign</referenceTo>
         <relationshipLabel>NPSP Data Imports (custom_campaign)</relationshipLabel>
```

Comment:
@davidmreed does this line needs to be updated too?

Reply:
No, the relationship label is cosmetic only, and the target field doesn't appear to cause the issue.